From 8d7689dcb8ca5df6cfa575b6b446451cab42eff5 Mon Sep 17 00:00:00 2001
From: Danny Nold <dannynold@freescale.com>
Date: Thu, 26 Aug 2010 18:36:07 -0500
Subject: [PATCH 174/505] ENGR00126698 - EPDC fb: Pan API should not trigger
 full screen update

Now that panning does not automatically trigger a full screen update,
the unit test must specify a full screen update after each pan
operation.

Signed-off-by: Danny Nold <dannynold@freescale.com>
---
 test/mxc_fb_test/mxc_epdc_fb_test.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/test/mxc_fb_test/mxc_epdc_fb_test.c b/test/mxc_fb_test/mxc_epdc_fb_test.c
index f2d5d8a..14c98ae 100644
--- a/test/mxc_fb_test/mxc_epdc_fb_test.c
+++ b/test/mxc_fb_test/mxc_epdc_fb_test.c
@@ -744,6 +744,7 @@ static int test_pan(void)
 			printf("Pan fail!\n");
 			break;
 		}
+		update_to_display(0, 0, screen_info.xres, screen_info.yres, WAVEFORM_MODE_AUTO, TRUE);
 	}
 	printf("Pan test done.\n");
 
-- 
1.8.0

